WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -264,7 +259,6 @@ - (UINavigationController *)notificationsNavigationController
     _notificationsNavigationController.tabBarItem.selectedImage = notificationsTabBarImage;
     _notificationsNavigationController.tabBarItem.imageInsets = [self tabBarIconImageInsets];
     _notificationsNavigationController.restorationIdentifier = WPNotificationsNavigationRestorationID;
-    _notificationsNavigationController.tabBarItem.title = @"";
```

Comment:
@astralbodies did we go back to not having labels on the tab bar? Because that's how it is right now.

I'm also noticing all these `tabBarItem`s are not consistent with `accessibilityLabel` and `accessibilityIdentifier`.